### ISSUE TITLE
How to compile

### ISSUE THREAD

**Issue by HardikSo** [ColinEberhardt/wasm-sudoku-solver#4]
Not opening in browser and how to compile this?

**Comment by ColinEberhardt** [comment]
There is no compilation required if you use a modern browser, you’ll need one that supports JavaScript imports: https://caniuse.com/#feat=mdn-javascript_statements_import

**Comment by HardikSo** [comment]
No, actually I want to make this project on my own that is why I was asking. Can you please put a readme for that?
And when I go to the site mentioned, in the console it gives the server responded with a status of 404 ()?

**Comment by HardikSo** [comment]
Okay , I got the camera working but it is not giving any solution to it.

**Comment by HardikSo** [comment]
the error it is throwing is "Cannot enlarge memory arrays. Either (1) compile with  -s TOTAL_MEMORY=X  with X higher than the current value 2147352576, (2) compile with  -s ALLOW_MEMORY_GROWTH=1  which allows increasing the size at runtime, or (3) if you want malloc to return NULL (0) instead of this abort, compile with  -s ABORTING_MALLOC=0 "

**Comment by elad-eyal** [comment]
I have this too on chrome

![image](https://user-images.githubusercontent.com/20379005/104811560-1bd5a880-5805-11eb-82a7-1b6fce0cea71.png)
